synapse/tests/test_model_orgs.py

```diff
@@ -485,6 +485,9 @@ async def test_ou_simple(self):
 
             iden = await core.callStorm('ou:id:type return($node.value())')
 
+            self.len(1, alts := await core.nodes('[ ou:id:type=({"name": "foobar"}) ]'))
+            self.eq(nodes[0].ndef, alts[0].ndef)
+
```

Comment:
We should add another couple lines that set the `:names` property on the node and uses it to deconflict based on an alts match :+1: